app/uk/gov/hmrc/excisemovementcontrolsystemapi/controllers/GetMessagesController.scala

```diff
@@ -48,7 +48,9 @@ class GetMessagesController @Inject()(
 
         movementService.getMatchingERN(lrn, request.erns.toList).flatMap {
           case None => Future.successful(BadRequest(Json.toJson(ErrorResponse(dateTimeService.timestamp(), "Invalid LRN supplied for ERN", ""))))
-          case Some(ern) => workItemService.addWorkItemForErn(ern, fastMode = false)
+            Future.successful(BadRequest(Json.toJson(ErrorResponse(dateTimeService.timestamp(), "Invalid LRN supplied for ERN", ""))))
```

Comment:
Line duplicated - guess it's a merge error?

Reply:
good catch. done